libraries/standard/mqtt/include/mqtt.h

```diff
@@ -128,6 +186,7 @@ typedef struct MQTTPubAckInfo
 } MQTTPubAckInfo_t;
 
 /**
+ * @ingroup mqtt_struct_types
  * @brief A struct representing an MQTT connection.
```

Comment:
The section for MQTTContext_t looks off since it shows "outgoingPublishRecords [(10U)]" instead of "outgoingPublishRecords [ MQTT_STATE_ARRAY_MAX_COUNT ]" as would be expected. Unsure if there's anything we can do about that

Reply:
Fixed to expand only PREDEFINED macros.